src/overlays/effects/ovl_Effect_Ss_Dust/z_eff_ss_dust.h

```diff
@@ -3,6 +3,16 @@
 
 #include "global.h"
 
+#define DUST_DRAWFLAG0 (0)
```

Comment:
I don't think we usually make defines for flags with value 0.

Reply:
Yes, though this time it actually has a meaning besides just NONE. It has its own type of drawing.

Reply:
To me it seems more like it's just not doing the things that the others do (although of course it's treating them more like an enum than actual flags since it's if-elses).

Reply:
The main reason I added it was for the function calls where the flags are just 4 (rand color offset), since I think it is clearer to or with this flag since the bottom three bits do act like an enum (even if they aren't for reasons).

Reply:
For more context here, I did try spawning dust that runs threw each drawflags intended branch (0, 1, 2). 0,2 look very similar if not identical (though could be a bad graphics pluggin), while 1 definitely looks different. Personally I still like `DRAWFLAG_0` since it still acts like an enum and has its own code path, even if it doesn't do much. Perhaps I don't call them flags but do something like 
```c
#define Dust_Draw0 0
#define Dust_Draw1 1
#define Dust_Draw2 2
```
since they aren't really flags, but also really an enum.